djangocms_blog/admin.py

```diff
@@ -82,6 +136,14 @@ class PostAdmin(PlaceholderAdminMixin, FrontendEditableAdminMixin,
     search_fields = ('translations__title',)
     date_hierarchy = 'date_published'
     raw_id_fields = ['author']
+    actions = [
```

Comment:
Please check this https://docs.djangoproject.com/en/1.10/ref/contrib/admin/actions/#actions-as-modeladmin-methods for how to implement method-based actions